server/middleware.ts

```diff
@@ -54,6 +54,14 @@ export async function middleware(request: NextRequest) {
     },
   );
 
+  if (process.env.NODE_ENV === "development" && process.env.BYPASS_AUTH === "true") {
+    const { data, error } = await supabase.auth.signInWithPassword({
```

Comment:
this may be worth putting in the config file too

Reply:
which part? the email/password for the demo user?